Sources/Datasets/datasets/image/MNIST.swift

```diff
@@ -57,6 +67,12 @@ public struct MNISTDataLoader: S5TFDataLoader {
                                       semaphore.signal()
                                     }
                 semaphore.wait()
+                do {
+                    try S5TFUtils.shell(URL(string: binaryLocation + "gunzip")!, ".s5tf-datasets/mnist/mnist_train_images")
+                }
```

Comment:
Consider using a URL for binaryLocation if possible.

I think `".s5tf-datasets/mnist/mnist_train_images"` should be determined by the URL because it might change in the future (unlikely but possible).